requirements.txt

```diff
@@ -1,2 +1,2 @@
-discord.py[voice]
+discord
```

Comment:
While the pip package name does not match the namespace, we should keep it as `discord.py[voice]` (since voice support isn't included on the `discord` package).

Just ignore the PyCharm warnings ;-)